libs/utils/decorate.js

```diff
@@ -260,3 +262,25 @@ export function applyInViewPortPlay(video) {
     observer.observe(video);
   }
 }
+
+export function turnAnchorIntoVideo({ hash, src, anchorTag }) {
+  const { dataset, parentElement } = anchorTag;
+  const attrs = getVideoAttrs(hash, dataset);
+  const video = `<video ${attrs}></video>`;
+  anchorTag.insertAdjacentHTML('afterend', video);
+  createIntersectionObserver({
+    el: parentElement,
+    options: { rootMargin: `${ROOT_MARGIN}px` },
```

Comment:
Seems `ROOT_MARGIN` is only used here, does it make sense to keep the value decoupled from the logic, since it's never re-used?

Reply:
Moved into the fn directly